### PR TITLE
SYS-172 Enforce MongoDB 3.2

### PR DESCRIPTION
**Pull request by zazabe**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'bundler', '~> 1.12.5'
+gem 'bundler', '~> 1.14.6'
```
Comment (ppp0) [inline Gemfile]: Where you able to build with bundler 1.14.6 ? I vaguely remember there was a dependency somewhere deep in that requires ~>1.12 max

Comment (zazabe) [inline Gemfile]: let me try again with this version

**Comment by zazabe** [comment]
`bundle exec rake build` works fine with `bundler-1.14.6`
